.eslintrc.json

```diff
@@ -22,7 +22,10 @@
     },
     "ecmaVersion": "latest",
     "sourceType": "module",
-    "project": "./tsconfig.json"
+    "project": [
+      "./tsconfig.json",
+      "./tsconfig.eslint.json"
```

Comment:
See comment left in https://github.com/josh-leyshon/ozbargain/pull/20/files#diff-5053c0bda6e3ba2fcfa5bbe118ecf5c60fa9f099b78b3e6deb66e5679424f515